scripts/build.js

```diff
@@ -162,19 +162,24 @@ build.photo = function(data) {
 
 build.overlay_image = function(data) {
 
```

Comment:
This function is wrong. The semantic is not preserved. What happens if `type == 'exif'` and `exifHash == ''` ?

The behaviour I would suggest would be:
```js
let exifHash  = data.make + data.model + data.shutter + data.aperture + data.focal + data.iso;

// Get the stored setting for the overlay_image
let type = lychee.image_overlay_type;

let html = ``;
if(type && type === 'desc' && data.description !== '')
{
	html += lychee.html`<div id="image_overlay"><p>${ data.description }</p></div>`;
}
else if (exifHash !== '') // type == null or type === 'exif' is implied here
{
	html += lychee.html`
		<div id="image_overlay"><h1>$${ data.title }</h1>
		<p>${ data.shutter.replace('s','sec') } at ${ data.aperture.replace('f/','&fnof; / ') }, ${ lychee.locale['PHOTO_ISO'] } ${ data.iso }<br>
		${ data.focal } ${ (data.lens && data.lens !== '') ? '(' + data.lens+ ')' : ''}</p>
		</div>`;
}

return html;
```

Also make sure that the curly are placed correctly. I am quite sure this is not the case in this PR.